blueocean-dashboard/src/main/js/creation/github-enterprise/api/CreateServerError.js

```diff
@@ -6,7 +6,8 @@ class CreateServerError extends ServerError {
     _handleCustomErrors(errors) {
         this.duplicateName = errors.some(err => err.field === 'name' && err.code === 'ALREADY_EXISTS');
         this.duplicateUrl = errors.some(err => err.field === 'apiUrl' && err.code === 'ALREADY_EXISTS');
-        this.invalidUrl = errors.some(err => err.field === 'apiUrl' && err.code === 'INVALID');
+        this.invalidServer = errors.some(err => err.field === 'apiUrl' && err.code === 'INVALID' && err.message.indexOf('check hostname') !== -1);
+        this.invalidApiUrl = errors.some(err => err.field === 'apiUrl' && err.code === 'INVALID' && err.message.indexOf('check path') !== -1);
```

Comment:
We are depending on error message text to flag error, to protect against backend message change, better to have tests for it, ideally in ath.